### PR TITLE
Implement report changelogs to meetings

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

Much needed

Related PRs
https://github.com/webkom/lego/pull/3637
https://github.com/webkom/lego-editor/pull/1223

# Result

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

https://github.com/user-attachments/assets/2016abe5-7708-4524-8e94-de2f305135b2

# Testing

- [x] I have thoroughly tested my changes.

See video above

---

Resolves ABA-1061

### CODE REVIEW

File: app/routes/meetings/components/MeetingDetail.tsx [hunk 1]
```diff
@@ -108,6 +115,23 @@ const MeetingDetails = () => {
     }));
   };
 
+  const [changelogOpen, setChangelogOpen] = useState(false);
+  const [diff, setDiff] = useState<
+    { count: number; added: boolean; removed: boolean; value: string }[]
```
Comment (jonasdeluna) [inline app/routes/meetings/components/MeetingDetail.tsx:120]: Should this be a type instead?

**Review by jonasdeluna — APPROVED** [review]
Nice!

File: app/routes/meetings/components/MeetingDetail.tsx (hunk 1 above)
Comment (ivarnakken) [reply app/routes/meetings/components/MeetingDetail.tsx:120]: meh it's only used here 🤷🏼‍♂️

**Review by norbye — APPROVED** [review]

**Comment by ivarnakken** [comment]
Improved look for images and special html tags:
image

**Review by Bestem0r — APPROVED** [review]
Looks great! What do you think about adding `-` and `+` to the diffs like in git?

**Comment by ivarnakken** [comment]
> Looks great! What do you think about adding - and + to the diffs like in git?

I tried multiple designs and came to the conclusion that this was the cleanest. Nevertheless, that would have to be implemented in lego-editor😄

**Pull request merged**